index.js.flow

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+declare type ReadFunction<T> = (tag: number, result: T, pbf: Pbf) => void;
+
+declare class Pbf {
+    constructor(buf?: ArrayBuffer | Uint8Array): Pbf;
+
+    readFields<T>(readField: ReadFunction<T>, result: T, end?: number): T;
+    readMessage<T>(readField: ReadFunction<T>, result: T): T;
+
+    readFixed32(): number;
+    readSFixed32(): number;
+    readFixed64(): number;
+    readSFixed64(): number;
+    readFloat(): number;
+    readDouble(): number;
+    readVarint(): number;
+    readVarint64(): number;
+    readSVarint(): number;
+    readBoolean(): boolean;
+    readString(): string;
+    readBytes(): Uint8Array;
+}
+
+declare module.exports: typeof Pbf
```

Comment:
This only covers a fraction of the methods, mostly ones for reading, while there are also many for writing. We need to cover all the methods to be able to merge the PR.